board/samsung/board-c1s.c

```diff
@@ -6,9 +6,7 @@
 #include <board.h>
 #include <drivers/framework.h>
 #include <lib/simplefb.h>
-
-#define DECON_F_BASE		0x19050000
-#define HW_SW_TRIG_CONTROL	0x70
+#include <soc/exynos990.h>
```

Comment:
Ideally the soc directory will be dropped in the next ~10 commits. Might be better to just make board-990common or something like that. Also, I had an idea to implement a single common board for all exynoses and do chipid selection but I don't have all the devices.